### PR TITLE
Update webmaps

### PR DESCRIPTION
**Pull request by psykzz**
## About The Pull Request
The main purpose here was to add new web maps urls.

I looked at the configs for them and thought the data would be better suited for the json files.
Hosts can override these via configs still but the configs are not required.

**Addditionally**
This removes the maps from the game since byond doesn't support them
This updates the update_status world proc to be a little clearer.
This moves the webmap url to the json files, and allows server configs for overrides (but by default not required)

## Changelog
:cl:
add: Added more webmaps
del: SL maps are removed
code: webmap urls are now defined by the json files
/:cl:

### CODE REVIEW

**Comment by psykzz** [comment]
Forgot you don't build all maps by default, will cleanup the map items tomorrow.

File: _maps/bigred_v2.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "map_name": "Big Red",
     "map_path": "map_files/BigRed_v2",
-    "map_file": "BigRed_v2.dmm",
+	"map_file": "BigRed_v2.dmm",
```
Comment (spookydonut) [inline _maps/bigred_v2.json:4]: indents

File: _maps/ice_colony_v2.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {
     "map_name": "Ice Colony",
     "map_path": "map_files/Ice_Colony_v2",
-    "map_file": "Ice_Colony_v2.dmm",
+	"map_file": "Ice_Colony_v2.dmm",
```
Comment (spookydonut) [inline _maps/ice_colony_v2.json:4]: indents

File: _maps/icy_caves.json [hunk 3]
```diff
@@ -2,6 +2,7 @@
     "map_name": "Icy Caves",
     "map_path": "map_files/icy_caves",
     "map_file": "icy_caves.dmm",
+	"webmap_url": "https://bit.ly/tgmc-icycaves",
```
Comment (spookydonut) [inline _maps/icy_caves.json:5]: indents

File: _maps/lv624.json [hunk 4]
```diff
@@ -2,6 +2,7 @@
     "map_name": "LV624",
     "map_path": "map_files/LV624",
     "map_file": "LV624.dmm",
+	"webmap_url": "https://bit.ly/tgmc-lv624",
```
Comment (spookydonut) [inline _maps/lv624.json:5]: indents

File: _maps/pillar_of_spring.json [hunk 5]
```diff
@@ -2,6 +2,7 @@
     "map_name": "Pillar of Spring",
     "map_path": "map_files/Pillar_of_Spring",
     "map_file": "TGS_Pillar_of_Spring.dmm",
+	"webmap_url": "https://bit.ly/tgmc-pillar",
```
Comment (spookydonut) [inline _maps/pillar_of_spring.json:5]: indents

File: _maps/prison_station_fop.json [hunk 6]
```diff
@@ -2,6 +2,7 @@
     "map_name": "Prison Station",
     "map_path": "map_files/Prison_Station_FOP",
     "map_file": "Prison_Station_FOP.dmm",
+	"webmap_url": "https://bit.ly/tgmc-prison",
```
Comment (spookydonut) [inline _maps/prison_station_fop.json:5]: indents

File: _maps/research_outpost.json [hunk 7]
```diff
@@ -2,5 +2,6 @@
     "map_name": "Research Outpost",
     "map_path": "map_files/Research_Outpost",
     "map_file": "Research_Outpost.dmm",
+	"webmap_url": "https://bit.ly/tgmc-ro",
```
Comment (spookydonut) [inline _maps/research_outpost.json:5]: indents

File: _maps/sulaco.json [hunk 8]
```diff
@@ -2,5 +2,6 @@
     "map_name": "Sulaco",
     "map_path": "map_files/Sulaco",
     "map_file": "Sulaco.dmm",
+	"webmap_url": "https://bit.ly/tgmc-sulaco",
```
Comment (spookydonut) [inline _maps/sulaco.json:5]: indents

File: _maps/theseus.json [hunk 9]
```diff
@@ -2,5 +2,6 @@
     "map_name": "Theseus",
     "map_path": "map_files/Theseus",
     "map_file": "TGS_Theseus.dmm",
+	"webmap_url": "https://bit.ly/tgmc-thesues",
```
Comment (spookydonut) [inline _maps/theseus.json:5]: indents

File: _maps/whiskey_outpost_v2.json [hunk 10]
```diff
@@ -2,5 +2,6 @@
     "map_name": "Whiskey Outpost",
     "map_path": "map_files/Whiskey_Outpost",
     "map_file": "Whiskey_Outpost_v2.dmm",
+	"webmap_url": "https://bit.ly/tgmc-wo",
```
Comment (spookydonut) [inline _maps/whiskey_outpost_v2.json:5]: indents

File: code/game/objects/items/maps.dm [hunk 11]
```diff
@@ -1,104 +1,39 @@
...
-		if(MAP_ICE_COLONY)
-			new /obj/item/map/ice_colony_map(loc)
+/proc/get_ground_map_url(map_name = SSmapping.configs[GROUND_MAP].map_name)
+	var/current_map = length(SSmapping.configs) && SSmapping.configs[GROUND_MAP] ? SSmapping.configs[GROUND_MAP].map_name : null
+	var/default_ground_url = current_map && current_map == map_name ? SSmapping.configs[GROUND_MAP].webmap_url : null
+	var/ground_url
+	switch(map_name)
 		if(MAP_BIG_RED)
-			new /obj/item/map/big_red_map(loc)
+			ground_url = CONFIG_GET(string/webmap_url_ground_bigred)
```
Comment (spookydonut) [inline code/game/objects/items/maps.dm:7]: just pull from the map_config and remove the config, or remove it from the jsons and just use the config

**Pull request merged**